x-pack/plugins/search_indices/public/components/details_page/details_page.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiPageSection, EuiSpacer, EuiButton, EuiPageTemplate } from '@elastic/eui';
+import React, { useMemo } from 'react';
+import { FunctionComponent } from 'react';
```

Comment:
You can include this in the import above this, or if we want to just `import type {}` we could keep it separate.